concept/read_api_test.go

```diff
@@ -133,7 +140,7 @@ func TestGetConceptsByIDsUnmarshallingPayloadError(t *testing.T) {
 
 	ctx := tidUtils.TransactionAwareContext(context.Background(), tidUtils.NewTransactionID())
 	_, err := csAPI.GetConceptsByIDs(ctx, []string{"an-id"})
-	assert.EqualError(t, err, "invalid character '}' looking for beginning of value")
+	assert.Error(t, err)
```

Comment:
I suspect that the returned error is some form of json error. Something like `json.SintaxError` We can check the documentations. 
So we can test with `errors.As` for it.